parser.go

```diff
@@ -341,11 +341,11 @@ func parseFloat(frame Frame) (float64, Frame, *ParserError) {
 }
 
 func parseBool(frame Frame) (bool, Frame, *ParserError) {
-	if frame.end-frame.start >= 4 && frame.until(frame.start+4).str() == "true" {
-		return true, frame.forward(4), nil
+	if idx := strings.Index(strings.TrimSpace(frame.str()), "true"); idx == 0 {
+		return true, frame.forward(strings.Index(frame.str(), "true") + 4), nil
 	}
-	if frame.end-frame.start >= 5 && frame.until(frame.start+5).str() == "false" {
-		return false, frame.forward(5), nil
+	if idx := strings.Index(strings.TrimSpace(frame.str()), "false"); idx == 0 {
+		return false, frame.forward(strings.Index(frame.str(), "false") + 5), nil
 	}
 	return false, frame, newParserError(frame, "bool expected")
 }
```

Comment:
This part is already solved on the master branch by using skipWhiteSpaces which is more consistent with the rest of the parser. This is where the remaining conflict is.

Reply:
OK then this section will be ignored on merge